CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+## Contribution guidelines
+
+Contributions to `becquerel` are welcome and encouraged, whether it is
+reporting bugs, requesting features, or contributing code.
+Please follow these guidelines when contributing to this project.
+
+### Reporting issues
+
+When reporting issues with `becquerel`, please provide a minimum working example to help identify the problem and tag the issue as a `bug`.
+
+### Feature requests
+
+For a feature request, please create an issue and label it as a `new feature`.
+
+### Checklist for code contributions:
+  - [ ] Branch off of `develop` and name the branch `feature-XX` or `issue-XX`
```

Comment:
Does anyone have an opinion about branch naming conventions? I wrote this just as a proposal.

Reply:
I think the way you phrased it is good. I just didn't adhere to it so far.